### ISSUE TITLE
Complete documentation

### ISSUE THREAD

**Issue by PaulBone** [libfirm/sisyphus#2]
I've been looking for something like this, I have a shell script (which has a few bugs/edge cases).  This looks interesting but it lacks documentation, in particular any information on how to actually use it.

Thanks.

**Comment by brdjns** [comment]
Agreed. Basic usage information would be helpful, otherwise I can't really figure out how to use it.
